Generators/src/main/java/io/deephaven/pythonPreambles/plotV2.py

```diff
@@ -104,9 +104,9 @@ class Figure(JObjectWrapper):
 
     j_object_type = jpy.get_type("io.deephaven.plot.Figure")
 
-    def __init__(self, j_figure: jpy.JType = None):
+    def __init__(self, rows: int = 1, cols: int = 1, j_figure: jpy.JType = None):
```

Comment:
Should pydoc the new args.  @jmao-denver how are we doing that for constructors?

Reply:
If a wrapper should not be instantiated directly, we choose to omit the docstring for the initializer.  With this change, we should document it now and it will be a bit tricky for the j_figure parameter.

Reply:
I guess offering `figure(...)` may make sense, but I'm not sure without looking at the whole API.  It would break all of our existing docs.

Reply:
I could just add rows/cols as 3rd and 4th parameter instead, and then you can still specify it by adding the keywords, e.g. `Figure(rows=2, cols=2)`, so it's not a breaking change.
I just didn't think it made much sense API-wise having users pass in their own `j_figure` instance, seems like an anti-pattern?

Reply:
It does not make sense for users to pass in their own `j_figure`.  As is, your current change should not break any code.  I think the change @jmao-denver is proposing is adding a factory method, analogous to the methods that create `Table`s (e.g. `empty_table`).  If such a constructor method is added, it should be a package-level function named `figure` and should not be @classmethod.

I'm not sure what my thoughts are on pushing users to this constructor vs a package-level constructor function, but we need docs either way.

Reply:
My suggestions really just try to avoid documenting the j_figure parameter for the constructor. What I had in mind is a 'of' classmethod.
```
    def __init__(rows, cols) -> Figure:
         """ ... """

    @classmethod
    def of(j_figure) -> Figure:
        ..
```
then we change the generator code to use this new 'of' method instead of use the j_figure keyword parameter.

Reply:
We don't do that anywhere else.  The constructor still needs `j_figure`, so the extra complexity doesn't buy us anything and makes stuff more complex.  I'm fine if we don't document `j_figure` or document it as internal use only.

Reply:
As discussed we'll just update the constructor.